uniffi_bindgen/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "src/main.rs"
 
 [dependencies]
 cargo_metadata = "0.13"
-weedle = "0.12"
+weedle2 = "2.0.0"
```

Comment:
FWIW, I also think it's a reasonable move. I wonder if it might even be worth vendoring the fork of weedle into this repo rather than maintaining a separate `weedle2`, since I think in practice we would expect `uniffi` to be the only consumer of the fork.

Reply:
Not opposed to vendoring it, though then I would do that in a followup.